Sources/OutlineView/OutlineViewDataSource.swift

```diff
@@ -43,4 +89,151 @@ where Data.Element: Identifiable {
             return items[index]
         }
     }
+    
+    // MARK: - Drag & Drop
```

Comment:
You can move this into an extension within the same file instead of just a MARK comment.

Reply:
Oh. I didn't realize that. No worries then.